package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-json-files",
-  "version": "4.2.0",
+  "version": "4.2.1",
```

Comment:
not sure if this should be patch or minor

Reply:
This can happen outside the PR.